apps/condo/domains/ticket/components/TicketId/TicketUserInfoField.tsx

```diff
@@ -34,24 +34,34 @@ interface ITicketUserInfoFieldProps {
 export const TicketUserInfoField: React.FC<ITicketUserInfoFieldProps> = (props) => {
     const id = useMemo(() => get(props, ['user', 'id']), [props])
     const name = useMemo(() => get(props, ['user', 'name']), [props])
-    const nameLink = useMemo(() => get(props, ['nameLink'], '#'), [props])
+    const nameLink = useMemo(() => get(props, ['nameLink']), [props])
     const phone = useMemo(() => get(props, ['user', 'phone']), [props])
     const email = useMemo(() => get(props, ['user', 'email']), [props])
     const userInfo = useMemo(() => [], [])
```

Comment:
Could you please get rid of the `get` function call in this component at the same time? :)

Reply:
Do you propose to implement it in the following way?
```js
const nameLink = useMemo(() => props?.nameLink, [props])
```

Reply:
Yeah, optional chains

Reply:
Done that